eox-core-v-1-0/prose/edit/src/design-considerations-01-construction-principles.md

```diff
@@ -1,10 +1,32 @@
 ## Construction Principles
 
-TODO:
+An End-of-X (EOX) lifecycle statement represented by an OpenEoX Core document is created through the application of
+well-defined schemas during a process of collaboration between multiple stakeholders including vendors,
+Product Security Incident Response Teams (PSIRTs), and lifecycle management authorities.
+The focused and purposefully minimal schema is designed to be orchestrated with product identification schemas and
+put into context serving the goal of providing actionable, structured, and validated lifecycle information targeting a high degree of automation.
```

Comment:
Not part of the OpenEoX Core schema: The OpenEoX Core schema does just contain the standardized dates, its context provides the information about products that these dates apply for. This can be done through the neighbor specification OpenEoX Shell (as standalone) or the OpenEoX API or the integration into the product itself (e.g. via SNMP calls) or via integration into another standard (e.g. CSAF).